components/profile-controller/cmd/manager/main.go

```diff
@@ -33,7 +34,12 @@ import (
 func main() {
 	var metricsAddr string
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
+	var userIdHeader string
+	var userIdPrefix string
```

Comment:
Does the profile-controller run as a user?

Reply:
It has admin permission.
Here header and prefix are consistent across each platform.
Like service on GCP or on prem ldap will have same userIdHeader and prefix.